bricksrc/equipment.py

```diff
@@ -1,3 +1,4 @@
+from typing import Reversible
```

Comment:
Hey @carlosduarteroa this looks like something that was pulled in automatically by an IDE. Is this required by the implementation or can we remove it?

Reply:
I don't remember adding any additional packages and I think it is not necessary. We can remove it since I do use it. I wonder how it got there. I didn't know IDE could automatically pull in packages.

Reply:
Yeah, I noticed things like that happening when students were getting build errors in a class project because their IDE decided to be "helpful"

Reply:
I removed this line.